CHANGELOG.md

```diff
@@ -3,12 +3,17 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v1.0.0 - Kobbfarbad - [2024-03-20]
```

Comment:
what is the release name theme?

Reply:
swabian words, as suggested by @subwaystation :-)